src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Raven.Imports.Newtonsoft.Json;
+using Raven.Server.Commercial;
+using Raven.Server.Routing;
+
+namespace Raven.Server.Web.Studio
+{
+    public class LicenseHandler : RequestHandler
+    {
+        [RavenAction("/license/status", "GET")]
+        public Task Status()
+        {
+            HttpContext.Response.ContentType = "application/json";
+            HttpContext.Response.StatusCode = 200;
+            return HttpContext.Response.WriteAsync(JsonConvert.SerializeObject(Commercial.LicenseHandler.GetLicenseStatus()));
+        }
+
+        [RavenAction("/license/registration", "POST")]
+        public async Task Register()
+        {
+            var name = GetStringQueryString("name");
+            var email = GetStringQueryString("email");
+            var company = GetStringQueryString("company", false);
+            await Commercial.LicenseHandler.Register(new RegisteredUserInfo
```

Comment:
This doesn't actually _register_ anything. It just validate the license, nothing more. 
Why is the process split into register / activate?